crates/bevy_asset/src/server/mod.rs

```diff
@@ -963,6 +920,13 @@ pub struct MissingAssetLoaderForTypeNameError {
     type_name: String,
 }
 
+/// An error that occurs when an [`AssetLoader`] is not registered for a given [`Asset`] [`TypeId`].
+#[derive(Error, Debug)]
```

Comment:
```suggestion
#[derive(Error, Debug, PartialEq)]
```

PartialEq is really helpful on public error types to let people write tests.